app/(entrance)/sign-up/components/SignUpPage.js

```diff
@@ -180,35 +135,39 @@ export default function SignUpPage() {
     phone,
     zip,
     password,
-    candidateFirstName,
-    candidateLastName,
   } = state
 
-  const facilitatedSignUpMode = signUpMode === SIGN_UP_MODES.FACILITATED
 
   const enableSubmit =
-    validUserNames(state) && isValidEmail(email) && isValidPassword(password)
+    validUserNames(state) && 
```

Comment:
Since we're not actually validating usernames with this function anymore, we can just replace it with the existence check of first name and last name. 

`validUserNames(state)` -> `firstName && lastName`

That also means ln 110 can be deleted.